### PR TITLE
Document the release process more fully

### PR DESCRIPTION
**Pull request by davidfischer**
- Adds a changelog
- Adds more details on releasing

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+.. The text for the changelog is generated with ``npm run changelog``
+.. Then it is formatted and copied into this file.
+.. This is included by docs/changelog.rst
+
+
+Version v1.1.0
+---------------
+
+The major changes in this release were to send the client version with the ad request.
+In the future, we will begin warning users if their ad client is very out of date.
```
Comment (ericholscher) [inline CHANGELOG.rst:13]: Do we have a way to do this in the client? I wonder if we should add some kind of support for showing a warning from the API response, so we can actually alert the older clients in the next release.

File: docs/releasing.rst [hunk 2]
```diff
@@ -0,0 +1,34 @@
+:orphan:
```
Comment (ericholscher) [inline docs/releasing.rst:1]: Should we not link this from a toctree?

File: docs/releasing.rst [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  the latest version of the documentation.
+* Run ``npm run changelog`` and update ``CHANGELOG.rst``
+  (included by :doc:`/changelog`)
+  with the release date and details.
+* Commit these changes, create a pull request, and merge it.
+* Tag the release:
+  
+    .. code-block:: bash
+
+        export VERSION=vX.Y.Z
```
Comment (ericholscher) [inline docs/releasing.rst:21]: Seems unnecessary if we're only using it once in the script, but I guess it makes the version more explicit.

File: docs/releasing.rst [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+        git checkout main
+        git pull origin main
+        git tag $VERSION
+        git push --tags origin main
+
+* Release the `beta client`_ and purge the client from the CDN.
+  A few publishers (notably Read the Docs) use the beta client
+  and we can roll it out to verify no breaking changes before pushing this to third-party publishers.
+* Release the `release client`_ and purge the CDN.
+  After the release, the beta client and release client should be exactly the same.
```
Comment (ericholscher) [inline docs/releasing.rst:31]:
```suggestion
  After the release, the beta client and release client should be exactly the same.
  
.. note:: In the future we plan to release the client to NPM directly for users, but there is still a lot of churn and we don't want users pinning to old versions quite yet.
```

**Review by ericholscher — APPROVED** [review]
Looks good 👍

File: CHANGELOG.rst (hunk 1 above)
Comment (davidfischer) [reply CHANGELOG.rst:13]: That's my plan but it is not implemented.

File: docs/releasing.rst (hunk 2 above)
Comment (davidfischer) [reply docs/releasing.rst:1]: There's no toctree. It's using an implicit one that covers all the sections of the main document.

File: docs/releasing.rst (hunk 3 above)
Comment (davidfischer) [reply docs/releasing.rst:21]: Having configurable values not buried in the middle results in fewer copy/paste errors in my experience.

**Pull request merged**